pysat/tests/test_instruments.py

```diff
@@ -14,11 +14,22 @@
 
 saved_path = pysat.data_dir
 
+# Optional code to pass through user and password info to test instruments
+# dict, keyed by pysat instrument, with a list of usernames and passwords
+# user_info = {'platform_name': {'user': 'pysat_user',
+#                                'password': 'None'}}
+user_info = {'pysat_testing': {'user': 'pysat_user',
+                               'password': 'pysat_for_me'}}
```

Comment:
Should we have the developer email as the password and 'pysat_testing' as the user?  That way Madrigal will know what's going on and be able to contact us.

Reply:
I am using the the developer email over at `pysatMadrigal`. The tests here are just to test the testing instruments, so username and password are somewhat arbitrary.

Reply:
Yes, but it seems like an informative arbitrary example may be better.